### DISCUSSION TITLE
申请的sql工单，最后执行的时候一直显示是在排队中，是什么原因导致的？

### DISCUSSION THREAD

**Discussion by dingyansheng** [hhyo/Archery#2082] [General]
申请的sql工单，最后执行的时候一直显示是在排队中，是什么原因导致的？

**Comment by hhyo** [comment]
https://github.com/hhyo/archery/wiki/faq#sql%E4%B8%8A%E7%BA%BF%E7%9B%B8%E5%85%B3

先逐一排查
